### PR TITLE
Refactor the `perf` module

### LINKED ISSUE #183: evalscope 5.5 stream.日志不起作用
evalscope perf --url 'http://127.0.0.1:9600/v1/chat/completions' \
--parallel 128 --model 'Qwen2___5-32B-Instruct' --log-every-n-query 10 --read-timeout=360 \
--dataset-path '/home/ai/.cache/evalscope/open_qa.jsonl' -n 2048 --max-prompt-length 128000 \
--tokenizer-path '/home/ai/hub/Qwen2___5-32B-Instruct' \
--api openai --stop '<|im_end|>' --stream --dataset openqa

使用者命令 
image
一致在输出流式的结果。5.2 是可以的

### PR DESCRIPTION
**Pull request by Yunnglin**
- [x] Simplify CLI
- [x] Remove duplicate code
- [x] Remove not used code
- [x] Remove `pack_resouce` in `setup.py`, add `MANIFEST.in` 
- [x] add unittest
- [x] update docs

fix #183

### CODE REVIEW

File: evalscope/perf/README.md
Comment (wangxingjun778) [inline evalscope/perf/README.md:1]: Please add some examples for perf usage in the module `examples`.

File: docs/zh/user_guides/stress_test.md [hunk 1]
```diff
@@ -1,162 +1,342 @@
...
+- `--stream` 使用SSE流输出，默认为False。
+
+### Prompt设置
+- `--max-prompt-length` 最大输入prompt长度，默认为`sys.maxsize`，大于该值时，将丢弃prompt。
+- `--min-prompt-length` 最小输入prompt长度，默认为0，小于该值时，将丢弃prompt。
+- `--prompt` 指定请求prompt，一个字符串或本地文件，使用优先级高于`dataset`。使用本地文件时，通过`@/path/to/file`指定文件路径，例如`@./prompt.txt`。
+- `--query-template` 指定查询模板，一个`JSON`字符串或本地文件，使用本地文件时，通过`@/path/to/file`指定文件路径，例如`@./query_template.json`。
+
+### 数据集配置
+- `--dataset` 指定数据集[openqa|longalpaca|line_by_line]，您可以使用python定义自定义数据集解析器，并指定python文件路径，参考`dataset_plugin_base.py`。
```
Comment (wangxingjun778) [inline docs/zh/user_guides/stress_test.md]: 这里可以在下文中加一段文字，给到用户自定义数据集的pipeline，然后在这里引用，替换掉 `dataset_plugin_base.py`；

File: docs/en/user_guides/stress_test.md [hunk 2]
```diff
@@ -1,174 +1,352 @@
-# Stress Testing
-A stress testing tool that focuses on large language models and can be customized to support various data set formats and different API protocol formats.
-## Usage
+# Model Inference Performance Stress Test
 
-### Command line  
+A stress testing tool for large language models that can be customized to support various dataset formats and different API protocol formats, with default support for OpenAI API format.
+
+## Quick Start
```
Comment (wangxingjun778) [inline docs/en/user_guides/stress_test.md:11]: Add `Installation` section for perf

File: docs/en/user_guides/stress_test.md [hunk 3]
```diff
@@ -182,32 +360,98 @@ with con:
...
+|     6144      |     334.71      |      0.0       |
+|     14336     |     318.88      |      0.0       |
+|     30720     |     292.86      |      0.0       |
++---------------+-----------------+----------------+
+```
+
+## Custom Request API
+Currently supported API request formats are openai and dashscope. To extend the API, you can create a subclass of `ApiPluginBase` and use the `@register_api("api name")` decorator.
+
+You can build the request using `model`, `prompt`, and `query_template` in the `build_request` method. You may refer to `opanai_api.py`.
```
Comment (wangxingjun778) [inline docs/en/user_guides/stress_test.md]: opanai_api.py not found

File: docs/en/user_guides/stress_test.md (hunk 2 above)
Comment (Yunnglin) [reply docs/en/user_guides/stress_test.md:11]: done

File: docs/zh/user_guides/stress_test.md (hunk 1 above)
Comment (Yunnglin) [reply docs/zh/user_guides/stress_test.md]: done

File: .pre-commit-config.yaml
Comment (wangxingjun778) [inline .pre-commit-config.yaml:1]: Add news in README.md for `perf refactoring`

File: docs/en/user_guides/stress_test.md (hunk 3 above)
Comment (Yunnglin) [reply docs/en/user_guides/stress_test.md]: fixed

File: .pre-commit-config.yaml
Comment (Yunnglin) [reply .pre-commit-config.yaml:1]: done

File: evalscope/perf/README.md
Comment (Yunnglin) [reply evalscope/perf/README.md:1]: done

**Pull request merged**